File: C/1010.c
#include <stdio.h>

int main() {
    int cod_1, n1, cod_2, n2;
    double v1, v2;

    scanf("%d %d %lf", &cod_1, &n1, &v1);
    scanf("%d %d %lf", &cod_2, &n2, &v2);

    printf("VALOR A PAGAR: R$ %.2lf\n", (n1 * v1) + (n2 * v2));

    return 0;
}

File: C#/1010.cs
using System;
using System.Globalization;

class URI {

    static void Main(string[] args) { 
        string[] peca1 = Console.ReadLine().Split(' ');
        string[] peca2 = Console.ReadLine().Split(' ');

        int quantidade1 = int.Parse(peca1[1]);
        double valor1 = double.Parse(peca1[2], CultureInfo.InvariantCulture);

        int quantidade2 = int.Parse(peca2[1]);
        double valor2 = double.Parse(peca2[2], CultureInfo.InvariantCulture);

        double total = (quantidade1 * valor1) + (quantidade2 * valor2);

        Console.WriteLine("VALOR A PAGAR: R$ " + total.ToString("F2", CultureInfo.InvariantCulture));
    }

}